### PR TITLE
Add missing LocalNow channels.

### PR DESCRIPTION
**Pull request by heroku-miraheze**

### CODE REVIEW

File: channels/unsorted.m3u [hunk 1]
```diff
@@ -1027,6 +1029,7 @@ https://bloomberg-quicktake-1-pt.samsung.wurl.com/manifest/playlist.m3u8
 https://bloomberg-bloomberg-5-mx.samsung.wurl.com/manifest/playlist.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="" group-title="",Bloomberg TV
 https://liveprodapnortheast.akamaized.net/ap1/Channel-APTVqvs-AWS-tokyo-1/Source-APTVqvs-700-1_live.m3u8
+
```
Comment (freearhey) [inline channels/unsorted.m3u:1032]: Unnecessary blank line.

File: channels/unsorted.m3u [hunk 2]
```diff
@@ -1179,6 +1182,8 @@ https://veranda-butacatv-1-es.samsung.wurl.com/manifest/playlist.m3u8
 https://buzzr-roku-us.amagi.tv/hls/amagi_hls_data_buzzrAAAA-buzzr/CDN/playlist.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="" group-title="",Buzzr
 https://buzzr-roku-us.amagi.tv/playlistR404p.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="BUZZR" tvg-logo="https://f9q4g5j6.ssl.hwcdn.net/607724c904f7856e84332d15" group-title="Entertainment",BUZZR
+https://buzzr-localnow.amagi.tv/playlist.m3u8
```
Comment (freearhey) [inline channels/unsorted.m3u:1186]: Broken link.
Captura de pantalla 2021-04-28 a las 13 44 27

File: channels/unsorted.m3u [hunk 3]
```diff
@@ -963,6 +963,8 @@ https://59cba4d34b678.streamlock.net/canlitv/bitlistv/playlist.m3u8
 https://thegateway.app/BizAndYou/Biz_480p/playlist.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="https://i.imgur.com/3RUw9BW.png" group-title="",Black News Channel
 https://blacknewschannel-vizio.amagi.tv/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="Black News Channel" tvg-logo="https://f9q4g5j6.ssl.hwcdn.net/604270ac363bbc6ec7359778" group-title="News",Black News Channel
+https://linear-130.frequency.stream/dist/localnow/130/bpk-tv/BNC-OTT/default/index.m3u8
```
Comment (freearhey) [inline channels/unsorted.m3u:967]: The link is already in the playlist.

File: channels/unsorted.m3u [hunk 4]
```diff
@@ -1583,6 +1588,8 @@ https://live.chdrstatic.com/cbn/index.m3u8
 https://live.chdrstatic.com/cbn/primary/1.m3u8
 #EXTINF:-1 tvg-id="89217b" tvg-name="Cheddar_News" tvg-country="" tvg-language="" tvg-logo="https://i.imgur.com/aGcoJYU.png" group-title="",Cheddar News
 https://cheddar.plex.wurl.com/manifest/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="Cheddar News" tvg-logo="https://f9q4g5j6.ssl.hwcdn.net/607766d942e7e542aa73ddf2" group-title="News",Cheddar News
+https://livestream.chdrstatic.com/e1bced8988f6a21da2206247c69538b6d4b12898ae8245c3a72f57e21384ccba/localnow/localnowlive/cheddardigital/index.m3u8
```
Comment (freearhey) [inline channels/unsorted.m3u:1592]: The link is already in the playlist.

File: channels/unsorted.m3u [hunk 5]
```diff
@@ -2886,6 +2893,8 @@ https://outtv-froottv-1-eu.rakuten.wurl.com/manifest/playlist.m3u8
 http://stream.fs1.tv:8080/hls/webstream.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="https://www.samsung.com/us/smg/content/dam/s7/home/televisions-and-home-theater/tvs/tv-plus/all-channels/09092020/FTF_190x190.png?raw=true" group-title="",FTF For the fans
 https://elevensports.samsung.wurl.com/manifest/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="For The Fans" tvg-logo="https://f9q4g5j6.ssl.hwcdn.net/5f8d9952a1ac237d614cfba3" group-title="Sports",For The Fans
+https://1622862207.rsc.cdn77.org/FTF/FTF_SCTE.m3u8
```
Comment (freearhey) [inline channels/unsorted.m3u:2897]: Broken link.
Captura de pantalla 2021-04-28 a las 13 51 14

File: channels/unsorted.m3u [hunk 6]
```diff
@@ -4078,6 +4087,8 @@ http://5tv.lincoln.ne.gov/live/WIFI-2096k-1080p/WIFI-2096k-1080p.m3u8
 http://80tv.lincoln.ne.gov/live/WIFI-2096k-1080p/WIFI-2096k-1080p.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="" group-title="",LNKTV Health
 http://10tv.lincoln.ne.gov/live/WIFI-2096k-1080p/WIFI-2096k-1080p.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="LN Sports" tvg-logo="https://f9q4g5j6.ssl.hwcdn.net/5ff5007140095248f263f987" group-title="Sports",LN Sports
+https://d39jokgb64z0et.cloudfront.net/dist/localnow/86/hls/master/playlist.m3u8
```
Comment (freearhey) [inline channels/unsorted.m3u:4091]: The link is already in the playlist.

File: channels/unsorted.m3u [hunk 7]
```diff
@@ -4140,6 +4151,8 @@ https://d27r4t30huw0iy.cloudfront.net/playlist.m3u8
 https://bamus-spa-roku.amagi.tv/hls/amagi_hls_data_bamusaAAA-roku-bam-spanish/CDN/playlist.m3u8?rdid=8f35210d-be26-5e0d-8c57-d0f910026ee7&coppa=false&is_lat=1&content=3dde60b40c0c56758a7c4bf81df47691&genre=ROKU_ADS_CONTENT_GENRE&platform=web&content_type=livefeed&content_rating=ROKU_ADS_CONTENT_RATING&is_roku_lat=1
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="" group-title="",LSN
 https://asermedia-lacrossesportsnetwork-1-eu.rakuten.wurl.com/manifest/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="LSN" tvg-logo="https://f9q4g5j6.ssl.hwcdn.net/5f8edf2c18d68b4f7a59ed13" group-title="Sports",LSN
+https://1840769862.rsc.cdn77.org/FTF/LSN_SCTE.m3u8
```
Comment (freearhey) [inline channels/unsorted.m3u:4155]: The link is already in the playlist.

File: channels/unsorted.m3u [hunk 8]
```diff
@@ -4708,6 +4721,8 @@ https://lnc-news12.tubi.video/primary/5.m3u8
 https://livestream.chdrstatic.com/2272f9cc29d3f9e812a3b4cacbd767bf69c95a59434c3f482606a67962a0dd5e/localnow/n12nyLocalNow/news12newyork/primary/index.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="https://od.lk/s/MF8yMjQ0OTQ3NTlf/Tubi_LiveNews_840x840.png" group-title="",News 12 New York
 https://lnc-news12.tubi.video/index.m3u8
+#EXTINF:-1 tvg-id="(no tvg-id)" tvg-name="News 12 New York" tvg-logo="https://f9q4g5j6.ssl.hwcdn.net/5f9c57ad92cd1774a1091752" group-title="News",News 12 New York
+https://livestream.chdrstatic.com/2272f9cc29d3f9e812a3b4cacbd767bf69c95a59434c3f482606a67962a0dd5e/localnow/n12nyLocalNow/news12newyork/primary/index.m3u8
```
Comment (freearhey) [inline channels/unsorted.m3u:4725]: The link is already in the playlist.

File: channels/unsorted.m3u [hunk 9]
```diff
@@ -4728,6 +4743,8 @@ https://broadcaster1.prolivestream.net:8083/onair/newsnetweb/playlist.m3u8
 https://nrpus.bozztv.com/36bay2/gusa-newsnet/index.m3u8
 #EXTINF:-1 tvg-id="90331d" tvg-name="Newsmax" tvg-country="" tvg-language="" tvg-logo="https://i.imgur.com/a5zs7r7.png" group-title="",Newsmax
 https://newsmax-plex.amagi.tv/hls/amagi_hls_data_plexAAAAA-newsmax-plex/CDN/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="NewsMax" tvg-logo="https://f9q4g5j6.ssl.hwcdn.net/606f92e82172644613403a30" group-title="News",NewsMax
+https://nmxlocalnow.akamaized.net/hls/live/2020471/Live_1/index.m3u8
```
Comment (freearhey) [inline channels/unsorted.m3u:4747]: The link is already in the playlist.

File: channels/unsorted.m3u [hunk 10]
```diff
@@ -4770,6 +4787,8 @@ https://newsy-newsy-1.vizio.wurl.com/manifest/playlist.m3u8
 https://newsy.samsung.wurl.com/manifest/playlist.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="" group-title="",Newsy
 https://newsy.sinclair.wurl.com/manifest/1200.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="Newsy" tvg-logo="https://f9q4g5j6.ssl.hwcdn.net/5f96bdd939905e0ce7438dd4" group-title="News",Newsy
+https://547f72e6652371c3.mediapackage.us-east-1.amazonaws.com/out/v1/e3e6e29095844c4ba7d887f01e44a5ef/index.m3u8
```
Comment (freearhey) [inline channels/unsorted.m3u:4791]: The link is already in the playlist.

File: channels/unsorted.m3u [hunk 11]
```diff
@@ -5142,6 +5161,8 @@ https://outsidetvplus-stirr.amagi.tv/playlistR360p.m3u8
 https://vcp.myplaytv.com/oxigenotv/oxigenotv/playlist.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="OXIR TV" tvg-country="" tvg-language="" tvg-logo="https://i.imgur.com/q3cxxW3.jpg" group-title="",OXIR TV
 http://51.210.199.36/hls/stream.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="Pattrn" tvg-logo="https://f9q4g5j6.ssl.hwcdn.net/5f976cdcd40f5d6678647013" group-title="Documentary",Pattrn
+https://linear-76.frequency.stream/dist/localnow/76/hls/master/playlist.m3u8
```
Comment (freearhey) [inline channels/unsorted.m3u:5165]: The link is already in the playlist.

File: channels/unsorted.m3u [hunk 12]
```diff
@@ -7420,6 +7441,8 @@ https://vod365-sports-channel-network-1-gb.samsung.wurl.com/manifest/1200.m3u8
 https://playout4multirtmp.tulix.tv/live6/Stream1/playlist.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="" group-title="",Sports Grid
 https://sportsgrid-plex.amagi.tv/amRdirect/us_privacy=1---&did=3f68a808-1437-5e57-9eaf-7a12279d9462&dnt=0&X-Plex-Token=nkPyxTGm41qFS7TFC9Yh&channelId=5ef11485d33ab9004048a1ca/hls/amagi_hls_data_plexAAAAA-sportsgrid-plex/CDN/1280x720_2855600/index.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="SportsGrid" tvg-logo="https://f9q4g5j6.ssl.hwcdn.net/606f8d8b860cda4118158a19" group-title="Sports",SportsGrid
+https://sportsgrid-localnow-us.amagi.tv/playlist.m3u8
```
Comment (freearhey) [inline channels/unsorted.m3u:7445]: Broken link.
Captura de pantalla 2021-04-28 a las 13 54 12

File: channels/unsorted.m3u [hunk 13]
```diff
@@ -8900,6 +8923,8 @@ http://185.29.68.24/tvb.m3u8
 https://livestream.streamingconnect.com/hls/vtvweb.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="https://od.lk/s/MF8yMjg3MDIyMDBf/LocalNow_400x400.png" group-title="",VENN
 https://d80z5qf1qyhbf.cloudfront.net/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="VENN" tvg-logo="https://f9q4g5j6.ssl.hwcdn.net/5f9c298487761b0460464ad3" group-title="Entertainment",VENN
+https://venntv-localnow.amagi.tv/index.m3u8
```
Comment (freearhey) [inline channels/unsorted.m3u:8927]: The link is already in the playlist.

File: channels/unsorted.m3u [hunk 14]
```diff
@@ -9168,6 +9193,8 @@ https://whistle-1.vizio.wurl.com/manifest/playlist.m3u8
 https://whistlesports-tcl.amagi.tv/playlist.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="" group-title="",Whistle TV
 https://whistletv-roku.cinedigm.com/midroll/amagi_hls_data_xumo-host-whistlesports-roku/CDN/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="Whistle TV" tvg-logo="https://f9q4g5j6.ssl.hwcdn.net/607e342400bc556fa74b6bd2" group-title="Adventure",Whistle TV
+https://whistletv-localnow.amagi.tv/playlist.m3u8
```
Comment (freearhey) [inline channels/unsorted.m3u:9197]: Broken link.
Captura de pantalla 2021-04-28 a las 13 55 19

**Review by freearhey — CHANGES_REQUESTED** [review]

File: channels/unsorted.m3u (hunk 14 above)
Comment (LaneSh4d0w) [reply channels/unsorted.m3u]: Geolocked.

File: channels/unsorted.m3u (hunk 12 above)
Comment (LaneSh4d0w) [reply channels/unsorted.m3u]: Geolocked.
image

File: channels/unsorted.m3u (hunk 5 above)
Comment (LaneSh4d0w) [reply channels/unsorted.m3u]: Most certainly geolocked.

image

File: channels/unsorted.m3u (hunk 2 above)
Comment (LaneSh4d0w) [reply channels/unsorted.m3u]: Geolocked too.

**Comment by freearhey** [comment]
Since no changes have been made, I'm closing the PR for now.

**Pull request closed without merging**